pom.xml

```diff
@@ -511,7 +511,7 @@
       <dependency>
         <groupId>com.google.guava</groupId>
         <artifactId>guava</artifactId>
-        <version>16.0.1</version> <!-- note: version 17.0 breaks hadoop 2.6+ at runtime -->
+        <version>19.0</version> <!-- note: version 17.0 breaks hadoop 2.6+ at runtime -->
```

Comment:
I don't think we can make this change until Hadoop fixes the problem upstream.  Will try to find the relevant issue; I had a reference to it before.